src/main/java/com/example/staffmanagerapi/resource/CustomerResource.java

```diff
@@ -34,5 +36,10 @@ public ResponseEntity add(@RequestBody @Valid CustomerCreationDto customerDto) {
 
         return ResponseEntity.status(201).build();
     }
+
+    @GetMapping 
+    public ResponseEntity<GetCustomersOutDto> getCustomers() {
+        return ResponseEntity.status(200).body(this.customerService.getCustomers());
```

Comment:
tu peut utiliser le shortcut si tu veut `ResponseEntity.ok(this.cutomerService.getCustomers())`